client/ayon_resolve/plugins/publish/collect_editorial_package.py

```diff
@@ -1,88 +1,51 @@
+        media_pool_item = instance.data["transientData"]["timeline_pool_item"]
 
-            publish_data = data["publish"]
+        # get version from publish data and rise it one up
+        version = instance.data.get("version")
+        if version is not None:
+            version += 1
 
-            # get version from publish data and rise it one up
-            version = publish_data.get("version")
-            if version is not None:
-                version += 1
-
-                # make sure last version of product is higher than current
-                # expected current version from publish data
-                folder_entity = ayon_api.get_folder_by_path(
-                    project_name=project_name,
-                    folder_path=publish_data["folderPath"],
-                )
-                last_version = ayon_api.get_last_version_by_product_name(
-                    project_name=project_name,
-                    product_name=publish_data["productName"],
-                    folder_id=folder_entity["id"],
-                )
-                if last_version is not None:
-                    last_version = int(last_version["version"])
-                    if version <= last_version:
-                        version = last_version + 1
-
-                publish_data["version"] = version
-
-            publish_data.update(
-                {
-                    "mediaPoolItem": media_pool_item,
-                    "item": media_pool_item,
-                }
+            # make sure last version of product is higher than current
+            # expected current version from publish data
+            folder_entity = ayon_api.get_folder_by_path(
+                project_name=project_name,
+                folder_path=instance.data["folderPath"],
+            )
+            last_version = ayon_api.get_last_version_by_product_name(
+                project_name=project_name,
+                product_name=instance.data["productName"],
+                folder_id=folder_entity["id"],
             )
+            if last_version is not None:
+                last_version = int(last_version["version"])
+                if version <= last_version:
+                    version = last_version + 1
 
-            if publish_data.get("taskId"):
-                task_entity = get_task_by_id(
-                    project_name=project_name,
-                    task_id=publish_data["taskId"],
-                )
-                publish_data["taskEntity"] = task_entity
-                publish_data["task"] = task_entity["name"]
+            instance.data["version"] = version
 
-            instance.data.update(publish_data)
+        instance.data.update(
+            {
+                "mediaPoolItem": media_pool_item,
+                "item": media_pool_item,
+            }
+        )
 
-            self.log.info(f"Editorial Package: {instance.data}")
+        self.log.info(f"Editorial Package: {instance.data}")
```

Comment:
Pretty sure this should be debug log since it's not artist-facing information.

```suggestion
        self.log.debug(f"Editorial Package: {instance.data}")
```